drivers/serial/Kconfig.nrfx

```diff
@@ -72,9 +89,89 @@ config UART_0_NRF_RTS_PIN
 	help
 	  The GPIO pin to use for RTS.
 
-endif # UART_0_NRF_UART
+endif # UART_0_NRF_UART || HAS_HW_NRF_UARTE0
+
+# ----------------- port 1 -----------------
+config UART_1_NRF_UARTE
+	bool "nRF UARTE 1"
+	depends on HAS_HW_NRF_UARTE1
+	select NRF_UARTE_PERIPHERAL
+	help
+	  Enable nRF UART with EasyDMA on port 1.
+
+if UART_1_NRF_UARTE
+
+config UART_1_INTERRUPT_DRIVEN
+	bool "Enable UART Interrupt support on port 1"
+	depends on UART_INTERRUPT_DRIVEN
+	default y
+	help
+	  This option enables UART interrupt support on port 1.
+
+config UART_1_NRF_PARITY_BIT
+	bool "Enable parity bit"
+	help
+	  Enable parity bit.
+
+config UART_1_NRF_FLOW_CONTROL
+	bool "Enable flow control"
+	help
+	  Enable flow control. If selected, additionally two pins, RTS and CTS
+	  have to be configured.
+
+config UART_1_NRF_TX_PIN
+	int "TX Pin Number"
+	range 0 47 if SOC_NRF52840_QIAA
+	range 0 31
+	help
+	  The GPIO pin to use for TX.
+
+config UART_1_NRF_RX_PIN
+	int "RX Pin Number"
+	range 0 47 if SOC_NRF52840_QIAA
+	range 0 31
+	help
+	  The GPIO pin to use for RX.
+
+config UART_1_NRF_CTS_PIN
+	int "CTS Pin Number"
+	range 0 47 if SOC_NRF52840_QIAA
+	range 0 31
+	depends on UART_1_NRF_FLOW_CONTROL
+	help
+	  The GPIO pin to use for CTS.
+
+config UART_1_NRF_RTS_PIN
+	int "RTS Pin Number"
+	range 0 47 if SOC_NRF52840_QIAA
+	range 0 31
+	depends on UART_1_NRF_FLOW_CONTROL
+	help
+	  The GPIO pin to use for RTS.
+
+endif # UART_1_NRF_UARTE
+
+if NRF_UARTE_PERIPHERAL
+
+config UART_NRFX_RAM_BUFFER_SIZE
+	int "Size of RAM buffer"
+	default 32
+	help
+	  SPIM peripherals cannot transmit data directly from flash. Therefore,
```

Comment:
Looks like a copy-paste failure from the nrfx SPI driver.

Reply:
fixed